ppdc/genstrings.cxx

```diff
@@ -52,31 +52,31 @@ main(void)
   ppdcSource::add_include("../data");
 
   src     = new ppdcSource("sample.drv");
-  catalog = new ppdcCatalog(NULL);
+  catalog = new ppdcCatalog(nullptr);
```

Comment:
"nullptr" isn't always available on older platforms. Please stick with NULL (this code is one release away from being purged...)